### PR TITLE
Release 5.2.0

### PR DESCRIPTION
**Pull request by mj**
#### Pull Request (PR) description

Following the instructions on https://voxpupuli.org/docs/releasing_version/ I have prepared a new release of `puppet-telegraf`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.1.1](https://github.com/voxpupuli/puppet-telegraf/tree/v5.1.1) (2023-09-18)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-telegraf/compare/v5.1.0...v5.1.1)
+
+**Implemented enhancements:**
+
+- Add Puppet 8 support [\#214](https://github.com/voxpupuli/puppet-telegraf/pull/214) ([bastelfreak](https://github.com/bastelfreak))
```
Comment (smortex) [inline CHANGELOG.md:13]: Because of the new "feature", this should be a minor version bump.

File: metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-telegraf",
-  "version": "5.1.1-rc0",
+  "version": "5.1.1",
```
Comment (smortex) [inline metadata.json]: This should be 5.2.0. (MAJOR.MINOR.PATCH, we bump minor)

Please update the `metadata.json` file, re-run `bundle exec rake release:prepare` review and add the updated CHANGELOG.md, `git commit --amend` and `git push -f` to update the PR.

**Review by smortex — COMMENTED** [review]
Looks good, but this should be 5.2.0.  Can you update the PR accordingly?

File: metadata.json (hunk 2 above)
Comment (mj) [reply metadata.json]: I have updated the PR with version `5.2.0`.

**Review by smortex — APPROVED** [review]

**Comment by smortex** [comment]
Well done @mj! Release action triggered:
https://github.com/voxpupuli/puppet-telegraf/actions/runs/6232677973

… and release available:
https://forge.puppetlabs.com/modules/puppet/telegraf/readme

Thank you!

**Pull request merged**